docs/src/main/sphinx/installation/cli.rst

```diff
@@ -80,6 +80,14 @@ The detailed behavior is as follows:
   authentication system configuration.
 * Expired tokens force you to log in again.
 
+.. _cli-jwt-auth:
+
+JWT authentication
+^^^^^^^^^^^^^^^^^^
+
+Use the ``--access-token=<token>`` option To pass a Base64-encoded :doc:`JWT
+</security/jwt>` access token to the server. 
```

Comment:
Avoid wrapping links as it makes them harder to read

Reply:
OK